cupy/_core/core.pyx

```diff
@@ -2330,6 +2330,8 @@ cpdef tuple assemble_cupy_compiler_options(tuple options):
                 else:
                     # Unsupported CUDA 12.x variant
                     _bundled_include = None
+            elif major == 13:
```

Comment:
I assume out of scope, but is the previous check correct? Wouldn't 12.9 be excluded?